.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,19 @@
-Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.
+Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.
 
 * [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
 * [ ] This PR adds a new recipe.
 * [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
 * [ ] This PR updates an existing recipe.
 * [ ] This PR does something else (explain below).
+
+Everyone has access to the following BiocondaBot commands, which can be given in a comment:
+
+ * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
```

Comment:
Does the bot know not react to `@BiocondaBot command` if it's part of a code block?

Reply:
Nope, a comment has to start with `@BiocondaBot` or `@bioconda-bot`

Reply:
Perfect! So it wouldn't go berserk on every repost of this template; that's good ;).

Reply:
Thankfully not :)